Settings/sparkle_settings.ini

```diff
@@ -7,7 +7,7 @@ solution_verifier = NONE
 extractor_cutoff_time = 60
 
 [configuration]
-budget_per_run = 600
+wallclock_time = 600
```

Comment:
Maybe we can specify the CPU time and solver call budget here too, as an example for the user how to use it. Do you have any values we could plug in here that don't affect the results (too much)?